calendar-bundle/src/Resources/contao/dca/tl_calendar.php

```diff
@@ -250,7 +250,18 @@
 			'inputType'               => 'checkbox',
 			'eval'                    => array('tl_class'=>'w50'),
 			'sql'                     => "char(1) NOT NULL default ''"
-		)
+		),
+		'breadcrumbMode' => array
+		(
+			'label'                   => &$GLOBALS['TL_LANG']['tl_calendar']['breadcrumbMode'],
+			'default'                 => \Contao\CalendarModel::BREADCRUMB_MODE_OVERRIDE,
+			'exclude'                 => true,
+			'inputType'               => 'select',
+			'options'                 => CalendarModel::BREADCRUMB_MODES,
+			'reference'               => &$GLOBALS['TL_LANG']['tl_news_archive']['breadcrumbModes'],
```

Comment:
We should probably move this to the `default.xlf` file of the core bundle, so we can re-use it. (Also you are referring to `tl_news_archive` here.)

Reply:
Obsolute now, because I removed setting as discuessed.